docs/examples/viz_emwave_animation.py

```diff
@@ -0,0 +1,184 @@
+"""
+===============================================
+Electromagnetic Wave Propagation Animation
+===============================================
+
+A linearly polarized sinusoidal electromagnetic wave, propagating in the
+direction +x through a homogeneous, isotropic, dissipationless medium,
+such as vacuum. The electric field (blue arrows) oscillates in the
+±z-direction, and the orthogonal magnetic field (red arrows) oscillates in
+phase with the electric field, but in the ±y-direction.
+
+Function of the sinusoid used in the animation = sin(k*x - w*t + d)
+Where, k:wavenumber, x:abscissa, w:angular frequency, t:time, d:phase angle
+
+Importing necessary modules
+"""
+
+from fury import window, actor, utils, ui
+import numpy as np
+import itertools
+
+###############################################################################
+# function that updates and returns the coordinates of the waves which are
+# changing with time
+
+
+def update_coordinates(wavenumber, ang_frq, time, phase_angle):
+    x = np.linspace(-3, 3, npoints)
+    y = np.sin(wavenumber*x - ang_frq*time + phase_angle)
+    z = np.array([0 for i in range(npoints)])
+    return x, y, z
+
+###############################################################################
+# Variable(s) and their description-
+# radius_waves: radius of the points that will be used to construct the waves
+#         (default = 0.02)
+# npoints: For high quality rendering, keep the number of npoints high and
+#          the radius_waves small but kindly note that higher values for
+#          npoints will slow down the rendering process
+#          (default = 800)
+<<<<<<< HEAD:docs/examples/viz_emwave_animation.py
```

Comment:
These lines weren't there when I pushed, should I remove them? Or are they normal? @Nibba2018 @skoudoro ?